packages/jest-util/src/FakeTimers.ts

```diff
@@ -356,8 +353,10 @@ export default class FakeTimers<TimerRef> {
 
   _createMocks() {
     const fn = (impl: Function) =>
+      // @ts-ignore TODO: figure out better typings here
       this._moduleMocker.fn().mockImplementation(impl);
 
+    // TODO: add better typings; these are mocks, but typed as regular timers
```

Comment:
the two comments are connected. cc @SimenB

Reply:
I wanna remove the autospying anyways (I've removed it in the Lolex implementation)

Reply:
However, we should be able to infer the type of the function passed in (can do later)